src/plugins/performanceIndicator/pluginSpec.js

```diff
@@ -61,25 +58,21 @@ describe('the plugin', () => {
         expect(performanceIndicator).toBeDefined();
     });
 
-    it('correctly calculates fps', () => {
-        return countFramesPromise.then((frames) => {
-            expect(performanceIndicator.text()).toEqual(`${frames} fps`);
-        });
+    it('correctly calculates fps', async () => {
```

Comment:
So I think the problem here is that this test (which was written by me) is conceptually flawed. It's trying to validate that the performance indicator is showing the _correct_ fps, and there's no reliable way to do this from within a Jasmine test, so entire premise is flawed, and I should never have written it this way.

Instead I think that we should check that it shows any positive number (ie. not just the default "~ fps", and not "0 fps" because that suggests a regression in the code itself?). So we should parse the indicator text, and check that the bit before the string `fps` is a positive number and call it a day.

Trying to check that it's a specific number is subject to the vagaries of the test environment it's running on and what else is going on at the time.

Reply:
@akhenry so we change the test name to 'calculates a positive integer value between 1 and 1000' ?

Reply:
sounds good. I'll just test for any positive number

Reply:
@akhenry alrighty, pushed the updated test, checking only for a positive fps